docs/src/solvers/IntegralSolvers.md

```diff
@@ -12,10 +12,12 @@ The following algorithms are available:
   - `CubaDivonne`: Divonne from Cuba.jl. Requires `using IntegralsCuba`. Works only for `>1`-dimensional integrations.
   - `CubaCuhre`: Cuhre from Cuba.jl. Requires `using IntegralsCuba`. Works only for `>1`-dimensional integrations.
   - `GaussLegendre`: Uses Gauss-Legendre quadrature with nodes and weights from FastGaussQuadrature.jl.
+  - `QuadratureFunction`: Accepts a user-defined function that returns nodes and weights.
```

Comment:
I'm not so sure of the naming on this, and also it sounds more like a function than an algorithm. Maybe `GeneralQuadrature`?

Reply:
That's a repo with 0 stars, I'm not worried about people knowing that. I'm more worried that people know `ODEFunction`, `SDEFunction`, `NonlinearFunction`, `BVPFunction`, etc. as a standard in SciML for the SciMLFunction interface https://docs.sciml.ai/SciMLBase/stable/interfaces/SciMLFunctions/, but here it's breaking the rule. 

I think `QuadratureRule` sounds good.